packages/svelte/src/index-client.js

```diff
@@ -161,6 +161,32 @@ export function afterUpdate(fn) {
 	init_update_callbacks(component_context).a.push(fn);
 }
 
+/**
+ * The `onFrame` function schedules a callback to run on `requestAnimationFrame`. It must be called inside an effect (e.g. during component initialisation).
+ *
+ * `onFrame` does not run inside [server-side components](https://svelte.dev/docs/svelte/svelte-server#render).
+ *
+ * @template T
+ * @param {() => NotFunction<T> | Promise<NotFunction<T>> | (() => any)} fn
+ * @returns {void}
+ */
+export function onFrame(fn) {
```

Comment:
I've been thinking more about this and I think its the name I don't like. Maybe we could call it `onAnimationFrame` as this is really where you're meant to handle scheduling of animation related things. I worry about `onFrame` as it feels very close to `onMount` in that people will just want something to happen on the next frame, but in reality it continuously happens.